javascript/src/test/components/ChannelMessages/MessageScrollable.test.js

```diff
@@ -268,4 +268,53 @@ describe("MessageScrollableView tests", () => {
         }, 500)
     });
 
+    test("Messages sorted in chronological order", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const exampleMessage = [
+            {
+                "type": "message3",
+                "subtype": "channel_join",
+                "ts": "1611257072.000900", //Jan 21, 2021, 11:24:32
+                "user": "U017218J9B3",
+                "text": "<!channel> This is an announcement",
+                "channel": "section-6pm",
+                "user_profile": {
+                    "real_name": "Test Person"
+                }
+            },
+            {     
+                "type": "message1",
+                "subtype": "channel_join",
+                "ts": "1611875002.014300", //Jan 28 2021, 15:03:22
+                "user": "U017218J9B3",
+                "text": "<!channel> This is an announcement",
+                "channel": "section-6pm",
+                "user_profile": {
+                    "real_name": "Test Person"
+                }
+            },
+            {
+                "type": "message2",
+                "subtype": "channel_join",
+                "ts": "1611867078.013800", //Jan 28, 2021, 12:51:18
+                "user": "U017218J9B3",
+                "text": "<!channel> This is an announcement",
+                "channel": "section-6pm",
+                "user_profile": {
+                    "real_name": "Test Person"
+                }
+            }
+        ]
+        const {getAllByText} = render(<MessageScrollableView messages={exampleMessage}/>);
+            const dates = getAllByText(/2021.{15}/);
+            const expectedDate1 = "2021-01-28 23:03:22";
```

Comment:
Quick change: As documentation for the future students for 156 can you put a comment here briefly describing why these times are different timezone than the ones you have commented above? Props to everyone on your team for figuring this issue out. Just say something along the lines like:

The timestamps in this test are in UTC because Jest tests and the Github tests use UTC timezone

Reply:
Just added this!